examples/matRad_example13_fitAnalyticalParticleBaseData.m

```diff
@@ -99,6 +100,9 @@
 pln.machine         = 'matRadBDLold';
 
 pln.numOfFractions  = 30;
+
+pln.withoutMachineFile  = 1;
```

Comment:
I don't really like this property, since apart from this specific use case, it has no meaning at all. My idea for a change would be to create a dummy machine file that does not contain the tabulated data, but all the metadata, as is done in line 144. I will comment further down.

Reply:
Property was removed and replaced with the mentionen creation of a dummy machine file.